app/src/processing/app/platform/ThinkDifferent.java

```diff
@@ -131,12 +131,51 @@ public void handleQuitRequestWith(QuitEvent event, QuitResponse response) {
 //      offscreen.setVisible(true);
 //    }
   }
-  
+
 
 //  public ThinkDifferent(Base base) {
 //    this.base = base;
 //  }
 
+  /**
+   * Sets a handler on an instance of {@link Application}, taking into account JVM version
```

Comment:
Should say "JDK version", but I don't want to make a new commit.

Reply:
No, JVM is correct.

Reply:
My thinking on this is that the JDK includes the JVM plus libraries and other misc. stuff. You could theoretically change package names in the JDK 1.8 class files to JDK11-compliant names (eg. change `com.apple.eawt` to `java.awt.desktop`), and then run them on an older JVM (subject to only non-native code, etc). So my view is that this is still a JDK difference and not a JVM difference. JVM, in my mind, includes bytecodes plus support, and doesn't include the libraries (with some exceptions, eg. invokedynamic implementation).

Reply:
To play devil's advocate with myself: Originally, "Java" was defined as a JVM plus libraries plus language, so in reality, I think the lines are kind of fuzzy anyway. I won't object to whatever's chosen, "JVM" or "JDK".